preprocessors/ner/ner.py

```diff
@@ -40,48 +39,48 @@
 CONTEXT_DIR = tempfile.mkdtemp(prefix=dir_prefix)
 IMAGE_DIR = tempfile.mkdtemp(prefix=dir_prefix)
 
-# path to the stanford ner models (https://nlp.stanford.edu/software/CRF-NER.shtml)
+# path to the stanford ner models
 STANFORD_JAR = '/app/stanford-ner/stanford-ner.jar'
 STANFORD_MODEL = '/app/stanford-ner/ner-model-english.ser.gz'
 
 
-"""
-Save a html contaning an image to a given location
-:my_html: the html to save
-:name: name of the to save
-:out_dir: directory to save the image to
-"""
 def save_image(my_html, name, out_dir):
+    """
+    Save a html contaning an image to a given location
+    :my_html: the html to save
+    :name: name of the to save
+    :out_dir: directory to save the image to
+    """
     path_ = os.path.abspath(f"{out_dir}")
     image_b64 = my_html.split(",")[1]
     binary = base64.b64decode(image_b64)
-    with open(f"{path_}/{name}.png","wb") as file:
+    with open(f"{path_}/{name}.png", "wb") as file:
         file.write(eval(str(binary)))
```

Comment:
Just noticed this...with the binary data coming from user input, calling `eval` basically just opens us to RCE. A quick local test shows the following works:
```suggestion
        file.write(binary)
```
On another note, is CLIPscore fine loading from a .png extension even if the encoding is different (e.g., JPEG)?

Reply:
I'm not sure about the encoding, the documentation, or code, don't mention anything about the encoding.